sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/ParameterAssignedToBase.cs

```diff
@@ -54,18 +72,52 @@ protected sealed override void Initialize(SonarAnalysisContext context)
                         c.ReportDiagnosticWhenActive(Diagnostic.Create(SupportedDiagnostics[0], left.GetLocation(), left.ToString()));
                     }
                 },
-                SyntaxKindsOfInterest.ToArray());
+                SyntaxKindOfInterest);
         }
 
+        private bool IsReadBefore(SemanticModel semanticModel, ISymbol parameterSymbol, TAssignmentStatementSyntax assignment)
+        {
+            //FIXME: REMOVE DEBUG
```

Comment:
There is still a `FIXME` to be removed :)

Reply:
This review is exactly the reason why I introduced FIXME concept :))